Makefile

```
@@ -35,9 +35,15 @@ REGISTRY?=gcr.io/k8s-staging-networking
 TAG?=$(shell echo "$$(date +v%Y%m%d)-$$(git describe --always --dirty)")
 # the full image tag
 IMAGE?=$(REGISTRY)/$(IMAGE_NAME):$(TAG)
+PLATFORMS?=linux/amd64,linux/arm64
```

```suggestion
PLATFORMS?=linux/amd64
```

default only to one platfor, this make it works locally and for multiplatform we can always set the variable